holoviews/core/util.py

```diff
@@ -967,7 +968,8 @@ def get_dynamic_item(map_obj, dimensions, key):
         key_offset = max([key-map_obj.cache_size, 0])
         key = map_obj.keys()[min([key-key_offset,
                                   len(map_obj)-1])]
-        el = map_obj.map(lambda x: x[key], ['DynamicMap'])
+        el = map_obj.map(lambda x: x[key], ['DynamicMap'],
+                         clone=False)
```

Comment:
Very minor nitpick: I don't see why ``clone=False`` needs to be on the next line in this case. Even if it is over 80 characters long it looks a bit weird...